### ISSUE TITLE
Ignore & disable ticket creation for  CC & BCC 

### ISSUE THREAD

**Issue by atanu2016** [osTicket/osTicket#5328]
How to prevent tickets from being created if the support email address is in the CC or BCC of an email?

Need help.

I am Currently using Osticket 1.14.1 version.

**Comment by JediKev** [comment]
@atanu2016 

Do you mean that someone CC's your support email address (not intending on the email going to the Helpdesk and creating a ticket) and since the email was CC'ed a copy of the email is in the inbox and it is fetched?

Cheers.

**Comment by ntozier** [comment]
People do that here all the time.  I'd love to see a Ticket filter (on CC only) so I can dump these requests.

**Comment by guidokuehler** [comment]
Out Table ost_osticket_session have currently millions of entries over the Years. 

Can we delete very old entries from this table without any negative Effects or data corruptions in OSTicket? 

Best Regards 
Guido

**Comment by JediKev** [comment]
@guidokuehler 

This is not the right place for that question and is not related to the Issue at hand. I suggest that you create a post on our [Forum](https://forum.osticket.com) and ask that question.

**Edit:**
To answer your question though, yes you can delete them with no negative side effects.

Cheers.

**Comment by atanu2016** [comment]
> @atanu2016
> 
> Do you mean that someone CC's your support email address (not intending on the email going to the Helpdesk and creating a ticket) and since the email was CC'ed a copy of the email is in the inbox and it is fetched?
> 
> Cheers.

Yes. There is a filter in OS ticket & @ntozier  mentioned , if it is a CC only filter , then it would be really helpful. 

Or if there is any way to disable auto response at least for CC or BCC tickets.
